server/game/cards/03_TWI/events/GuardingTheWay.ts

```diff
@@ -0,0 +1,35 @@
+import AbilityHelper from '../../../AbilityHelper';
+import { EventCard } from '../../../core/card/EventCard';
+import { KeywordName, WildcardCardType } from '../../../core/Constants';
+
+export default class GuardingTheWay extends EventCard {
+    protected override getImplementationId() {
+        return {
+            id: '2359136621',
+            internalName: 'guarding-the-way',
+        };
+    }
+
+    public override setupCardAbilities() {
+        this.setEventAbility({
+            title: 'Give Sentinel for the phase and optionally +2/+2 if you have initiative',
```

Comment:
nit, wording: "optional" for us means "you can choose not to," which is not the case here

Reply:
Would it be better to just model the card language? 

`Give Sentinel for the phase. If you have the initiative, also give that unit +2/+2 for this phase.`